src/installer/pkg/projects/Microsoft.NET.ILLink/Microsoft.NET.ILLink.pkgproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We must not introduce new pkgproj projects: https://github.com/dotnet/runtime/issues/75966. Please use a NuGet Pack task based project instead.